### DISCUSSION TITLE
HTTPS secure access to JtlReporter

### DISCUSSION THREAD

**Discussion by ganeshjoe** [ludeknovy/jtl-reporter#295] [Q&A]
Hi All,

Is it possible to setup jtlreporter as https secure site based on docker compose installation setup?

**Comment by ludeknovy** [comment]
Hi @ganeshjoe 
I guess there is, maybe this could help? 
https://jtlreporter.site/docs/guides/deployments/aws-ec2-ssl-configuration

**Reply by ludeknovy** [reply]
Like the general idea would be to have a reverse proxy set up.
